app/controllers/jobs_controller.rb

```diff
@@ -1,7 +1,11 @@
 class JobsController < ApplicationController
-  before_filter :authorize_admin!, only: [:new, :create, :destroy]
+  include CurrentProject
```

Comment:
All this situations have been addressed in the following PR: [#567](https://github.com/zendesk/samson/pull/567)